### PR TITLE
Enhancement: Enable and configure `concat_space` fixer

### PR DESCRIPTION
**Pull request by localheinz**
This pull request

- [x] enables and configures the `concat_space` fixer
- [x] runs `make coding-standards`

Follows #559.

💁‍♂️ For reference, see https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v3.4.0/doc/rules/operator/concat_space.rst.

### CODE REVIEW

File: .php-cs-fixer.php [hunk 1]
```diff
@@ -13,6 +13,9 @@
 
 $config->setRules([
     'array_indentation' => true,
+    'concat_space' => [
+        'spacing' => 'one',
```
Comment (localheinz) [inline .php-cs-fixer.php:17]: We could also use `none` here, but fewer files were modified when using `one`.

**Review by kamil-tekiela — APPROVED** [review]

File: .php-cs-fixer.php (hunk 1 above)
Comment (cmb69) [reply .php-cs-fixer.php:21]: I prefer `one`. :)

**Review by cmb69 — APPROVED** [review]
Thank you!

Maybe wait a bit for review of others.

**Comment by cmb69** [comment]
Thank you!

**Comment by localheinz** [comment]
Thank you, @cmb69 and @kamil-tekiela!

**Pull request merged**